verilog/formatting/tree_unwrapper.cc

```diff
@@ -892,6 +950,21 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeNode& node) {
       // else close-out current token partition?
       break;
     }
+    case NodeEnum::kConditionalStatement: {
+      CurrentTokenPartition()->Parent()->ApplyPreOrder([](verible::VectorTree<UnwrappedLine>& node) {
+        const auto& range = node.Value().TokensRange();
+        if ((range.back().TokenEnum() == yytokentype::TK_else) &&
+            (range.end()->TokenEnum() == yytokentype::TK_if)) {
```

Comment:
Dereferencing an `end()` iterator is usually suspicious, as it's usually one position past the last valid element, but I understand you're using the fact that these are iterators into a longer sequence.
No change requested for now.